app/views/providers/transactions/show.html.erb

```diff
@@ -10,33 +10,30 @@
       <%= content_tag(:caption, t('date.date_period', from: date_from, to: date_to), class: 'govuk-table__caption') if @bank_transactions.any? %>
       <thead class="govuk-table__head">
         <tr class="govuk-table__row">
-          <th class="govuk-table__header" scope="col">
-            <a
-              class="govuk-body select-clear-all"
-              href=""
-              data-copy-select="<%= t '.col_select_all' %>"
-              data-copy-clear="<%= t '.col_clear_all' %>">
-            </a>
+          <th
+            class="govuk-table__header select-clear-all"
+            scope="col"
+            data-copy-select="<%= t '.col_select_all' %>"
+            data-copy-clear="<%= t '.col_clear_all' %>">
           </th>
-          <th class="govuk-table__header sort header-sort-asc table-combine_right_if_narrow" scope="col">
+          <th data-sort-type="date" class="govuk-table__header sort header-sort-desc table-combine_right_if_narrow" scope="col">
             <span class="govuk-visually-hidden">Sort by </span>
-            <span class="aria-sort-description"><%= t('.col_date') %></span>
-            <span class="table-hidden_right_title"> &amp; <%= t('.col_description') %></span>
+            <span class="aria-sort-description"><%= t('.col_date') %></span><span class="table-hidden_right_title"> &amp; <%= t('.col_description') %></span><span class="sort-direction-indicator"><span class="screen-reader-sort-indicator govuk-visually-hidden"> (currently sorted from newest to oldest)</span></span>
           </th>
           <th data-sort-type="alphabetic" class="govuk-table__header sort" scope="col">
             <span class="govuk-visually-hidden">Sort by </span>
-            <span class="aria-sort-description"><%= t('.col_description') %></span>
+            <span class="aria-sort-description"><%= t('.col_description') %></span><span class="sort-direction-indicator"><span class="screen-reader-sort-indicator govuk-visually-hidden"></span></span>
```

Comment:
Very minor: The direction indicator span is an additional item. It would be clearer if this was on it's own line, rather than being tacked onto the end of the description span.

Reply:
I put it on the same line because I thought there might be some issues if there was whitespace between the two `<span>`s.  I can't remember why, but that might be a Monday morning thing.